sky/cli.py

```diff
@@ -1425,6 +1425,7 @@ def launch(
             '`sky spot launch`. `sky launch` supports a '
             'single task only.')
     task = task_or_dag
+    print(task)
```

Comment:
Unintended?

Reply:
Yeah... Some remnant of debug 😔 Thanks for point that out!